concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,29 @@
+from django.core.checks     import Error
+from django.core.checks     import register
+from django.core.checks     import Warning  # pylint: disable=redefined-builtin
+from django.conf            import settings
+
+
+@register()
+def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=unused-argument
+
+    if not isinstance(settings.CONCENT_FEATURES, list):
+        return [Error('The value of CONCENT_FEATURES setting is not a list', id = 'concent.E001')]
+
+    uknown_features = set(settings.CONCENT_FEATURES) - {"concent-api", "gatekeeper", "admin-panel"}
```

Comment:
`{"concent-api", "gatekeeper", "admin-panel"}` This can be created from AVAILABLE_CONCENT_FEATURES.keys(), and also used directly in if below (but a comment that you check for unknown features above if would be nice).

Reply:
Can't do that if I try:
from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
I get:
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

Reply:
That's because we're using `url()` and `include()` in `constants.py`. This could be avoided by using plain strings, though `AVAILABLE_CONCENT_FEATURES` would become more complex and less flexible...

Reply:
@cameel Fix for this is what I have described yesterday on slack, we discussed it with Jakub and as agreed with you lets turn concent_api into django app and load system_check in AppConfig ready() method.

Reply:
I know. I just wanted to explain why this happens because making it not use these functions is a more straightforward solution in my opinion. But making `concent_api` an app works too so let's continue with that.

Reply:
Done, concent_api is now django app included in INSTALLED_APPS, witch allow us to use
`from concent_api.constants  import AVAILABLE_CONCENT_FEATURES`.

Reply:
:+1: